src/main/java/cn/nukkit/network/process/DataPacketManager.java

```diff
@@ -66,10 +67,16 @@ public void registerDefaultProcessors() {
                 new SetPlayerGameTypeProcessor(),
                 new LecternUpdateProcessor(),
                 new MapInfoRequestProcessor(),
+                /*
+                 * Minecraft doesn't really use this packet any more, and the client can send it and play the music it wants,
+                 * even though it's of no interest to the gameplay.
+                 * The client isn't supposed to be able to broadcast a sound to the whole server.
+                 * @Zwuiix
                 new LevelSoundEventProcessor(),
                 new LevelSoundEventProcessorV1(),
                 new LevelSoundEventProcessorV2(),
-                new PlayerHotbarProcessor(),
+                */
+                //new PlayerHotbarProcessor(),
```

Comment:
> Because at the moment there are problems with the hotbar and the durability of the items on it and by putting it in a comment it allows to fix a bug without wanting it in the end. issue: #1657

frérot fini ta PR stp, c'est trop la merde ce bug du drura dans l'inv